packages/core/addon/templates/components/card-renderer.hbs

```diff
@@ -27,6 +27,8 @@ Ember Concurrency into the rendering of the card component here.
           <Isolated
             @card={{@card}}
             @mode={{this.mode}}
+            @cardId={{this.sanitizedCardId}}
+            @templateId={{this.sanitizedTemplateId}}
```

Comment:
I don't think this is necessary. in general we should not pass in additional card detail into the card renderer. it can figure everything it needs from `@card` no need to add extra coupling to it.

Reply:
So my thinking was the user might change the card name (when we add that capability), the styles would break, so it’s better to use the card id. The card id creates “template_adopted-card-id card-id” classes. If the card has to adopted parents, the template_adopted-card defaults to its own id. We use card-id class to style the card. If this turns out to be a template and another card adopts from it, the new card has classes “template_adopted-card-id card-id”. The card-id is there so that user can use it to make any custom changes.

I only implemented this for the isolated card and not for the embedded card. There was an issue with embedded card that I thought we can address if we decide we need the template styles in embedded card too, which normally shouldn’t be necessary since embedded card should just be a static snapshot of the isolated version.

Reply:
the card name comes form the id :-) it's just the id without the `local-hub::` but ultimately all cards will come with their css baked in. what we are doing is 100% temporary.

Reply:
also there is no need to pass it in. we should strive to pass as little as possible.